tests/test_foss_cli_help.py

```diff
@@ -0,0 +1,128 @@
+__doc__ = """Test the "help" text of the different sub commands of foss_cli"""
```

Comment:
I would completely delete this file since it is similar as testing the `click` framework and would require an update each time a character is changed in one of the help messages.

Reply:
It does NOT test the help-text. It only checks, that the switches/types are available for the command. I agree, that it may look like a little overkill - but it helped me a lot when testing calling the foss_cli from "outside" e.g. with python -m or foss_cli cmd_line. At least I knew  that it was working. I think the file will be helpful in refactoring - as entries like in foss_cli.py:764 d["IS_REQUEST_FOR_HELP"] = False may need some refactoring in the future.

Reply:
Ok.